pkg/controller/spd/indicator-plugin/manager.go

```diff
@@ -49,10 +52,10 @@ type IndicatorGetter interface {
 	GetIndicatorSpecChan() chan types.NamespacedName
 	GetIndicatorStatusChan() chan types.NamespacedName
 
-	// GetIndicatorSpec + GetIndicatorStatus
+	// GetSPDSpec + GetSPDStatus
 	// for indicator get functions, IndicatorUpdater will return a channel to obtain the merged results.
-	GetIndicatorSpec(_ types.NamespacedName) *apiworkload.ServiceProfileDescriptorSpec
-	GetIndicatorStatus(_ types.NamespacedName) *apiworkload.ServiceProfileDescriptorStatus
+	GetSPDSpec(_ types.NamespacedName) *apiworkload.ServiceProfileDescriptorSpec
```

Comment:
maybe it's better to keep `GetIndicatorSpec`, since the logic is for updating indicator spec (despite that the returned value is entire spec for spd)